src/orchestrator/orchestrator/orchestrator.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import codecs
+import datetime
+import logging
+from time import sleep
+import jsonpickle
+
+from learner.learner import learn_for_users
+from scraper.scraper import scrap
+from server.frontendstructs import Document
+from topicmodeller.topicmodeller import TopicModeller, classify
+
+
+logging.basicConfig(format=u'%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO, ilename='orchestrator.log')
+
+
+def _to_json(document):
+    # set unicode and pretty-print
+    jsonpickle.set_encoder_options('simplejson', indent=4, ensure_ascii=False)
+    return jsonpickle.encode(document)
```

Comment:
set_encoder maybe should not be called each time we call to_json. if it's global then we can do it  setupEnv, if not maybe there is additional optional params to .encode methode ?